dropwizard-migrations/src/test/java/io/dropwizard/migrations/AbstractMigrationTest.java

```diff
@@ -10,6 +12,7 @@
 
     static {
         ArgumentParsers.setTerminalWidthDetection(false);
+        SqlGeneratorFactory.getInstance().unregister(AddColumnGeneratorSQLite.class);
```

Comment:
Created a PR against Liquibase https://github.com/liquibase/liquibase/pull/756. Before the bug is fixed, we have to add this workaround.